src/connectors/numeric-selector/connectNumericSelector.js

```diff
@@ -160,6 +160,46 @@ export default function connectNumericSelector(renderFn, unmountFn) {
         return state.removeNumericRefinement(attributeName);
       },
 
+      getWidgetState(fullState, { state }) {
+        const currentRefinement = this._getRefinedValue(state);
+        if (
+          // Does the current state contain the current refinement?
+          (fullState.numericSelector &&
+            currentRefinement === fullState.numericSelector[attributeName]) ||
+          // Is the current value the first option / default value?
+          currentRefinement === options[0].value
+        ) {
+          return fullState;
+        }
+
+        if (currentRefinement || currentRefinement === 0)
+          return {
+            ...fullState,
+            numericSelector: {
+              ...fullState.numericSelector,
+              [attributeName]: currentRefinement,
+            },
+          };
+        return fullState;
```

Comment:
It's not easy to read without brackets to the condition, I would have written that way:

```js
  if (currentRefinement || currentRefinement === 0) {
    return {
      ...fullState,
      numericSelector: {
        ...fullState.numericSelector,
        [attributeName]: currentRefinement,
      },
    };
  }
    
  return fullState;
```

Reply:
Yes to brackets!

Reply:
But maybe could we add that as a rule in our eslint config and do that in another PR? :)

Reply:
I am in favor for enforcing brackets in every situation as a linting and formatting rule. Just because it removes those kind of discussion and does not create a dual rule where sometimes it gets written with brackets, sometimes without.

Same thing for arrow function only returning an object without brackets.

WDYT?

Reply:
-1 for adding a rule against 

```js
const s = () => ({});
```

we'd be forced to write them like 

```js
const s = () => {
  return {
  };
};
```

Which isn't really that valuable IMO. Requiring braces around an if and disallowing inline `if` is definitely +1 from me!

Reply:
> const s = () => {
>   return {
>   };
> };

But prettier will do this for you so you won't even see it or have to think about it.

Reply:
My issue was that it's longer to type